apps/src/dance/ai/utils/generateBlocks.ts

```diff
@@ -0,0 +1,8 @@
+import {BlockSvg, Workspace} from 'blockly';
+
+export const generateBlocks = (workspace: Workspace): [BlockSvg, BlockSvg] => {
```

Comment:
nit: can we rename this to `generateAiBlocks` or `generateAiEffectBlocks` - something more specific now that it's in its own file?